src/main/java/com/example/ShortnerURL/Controller/PingController.java

```diff
@@ -21,6 +25,7 @@ public SuccessResponseDto ping() {
                     .status("ok")
                     .build();
         } catch (Exception e) {
+            logger.error("Error");
```

Comment:
here we log the exception that has been caught in the catch. So that exception is in the variable `e` here.